lib/doorkeeper/oauth/token_introspection.rb

```diff
@@ -163,7 +163,7 @@ def valid_token?
 
       # RFC7662 Section 2.1
       def authorized_token_matches_introspected?
-        authorized_token.token == @token.token
+        authorized_token.token == @token&.token
```

Comment:
Cause `&.` operator is support from ruby ver. >= 2.3, so is this ok if ruby version is < 2.3?